### ISSUE TITLE
Clicking on a mutation raises error because "seq" is undefined for mutation records

### ISSUE THREAD

**Issue by eharkins** [matsengrp/olmsted#26]
This should either have the same functionality as clicking on a tree node, or serve a different function, or do nothing. The first of these options requires putting the sequence itself in each of these mutation records. The latter requires limiting the scope of the click event in vega or using logic to not dispatch the action in the click handler if the sequence is not defined

**Comment by metasoarous** [comment]
I don't think we really need anything to happen here, so perhaps the latter is the way to go.
